test/optimus/pr/test_optimus_PR.wdl

```diff
@@ -13,7 +13,10 @@ workflow TestOptimusPR {
   String expected_picard_metrics_hash
 
   # Optimus inputs
-  Array[Array[File]] fastq_inputs  # array of arrays of matched fastqs [ [r1, r2, i1], ... ]
+  Array[File] r1
+  Array[File] r2
+  Array[File] i1
```

Comment:
I think you'll have to make this optional if you want to test not passing in an `i1` file.  Does our testing infrastructure allow for testing a workflow against multiple input jsons / is this something we even want to do?

Reply:
It should allow this, yes. I'm not sure we'd want to put in this use case.